src/GraphQL/Wrappers/Violation/ViolationCollectionInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\graphql\GraphQL\Wrappers\Violation;
+
+/**
+ * Violation collection interface.
```

Comment:
Let's not repeat the class name in comments. What is a violation collection used for? "A collection of violations that can be triggered by a GraphQL mutation"